Fix FTBFS with poppler again, they will never be able to keep APIs

Thanks to Ubuntu people for that patch
---
 texk/web2c/pdftexdir/pdftoepdf.cc |    8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

Index: texlive-bin-2009~svn15596/texk/web2c/pdftexdir/pdftoepdf.cc
===================================================================
--- texlive-bin-2009~svn15596.orig/texk/web2c/pdftexdir/pdftoepdf.cc	2009-10-18 10:21:47.000000000 +0900
+++ texlive-bin-2009~svn15596/texk/web2c/pdftexdir/pdftoepdf.cc	2009-10-18 10:22:28.000000000 +0900
@@ -650,7 +650,7 @@
     }
     for (r = encodingList; r != 0; r = n) {
         n = r->next;
-        delete r->font;
+//        delete r->font;
         delete r;
     }
 }
@@ -690,6 +690,7 @@
     Page *page;
     int rotate;
     PDFRectangle *pagebox;
+    int minor_pdf_version_found, major_pdf_version_found;
     float pdf_version_found, pdf_version_wanted;
     // initialize
     if (!isInit) {
@@ -705,8 +706,11 @@
     // this works only for PDF 1.x -- but since any versions of PDF newer
     // than 1.x will not be backwards compatible to PDF 1.x, pdfTeX will
     // then have to changed drastically anyway.
-    pdf_version_found = pdf_doc->doc->getPDFVersion();
+    minor_pdf_version_found = pdf_doc->doc->getPDFMinorVersion();
+    major_pdf_version_found = pdf_doc->doc->getPDFMajorVersion();
+    pdf_version_found = major_pdf_version_found + (minor_pdf_version_found * 0.1);
     pdf_version_wanted = 1 + (minor_pdf_version_wanted * 0.1);
+
     if (pdf_version_found > pdf_version_wanted) {
         char msg[] =
             "PDF inclusion: found PDF version <%.1f>, but at most version <%.1f> allowed";
